tomcat/src/main/docker/50-tomcat.bash

```diff
@@ -5,6 +5,10 @@ if [ "$PLATFORM" == "gae" ]; then
   TOMCAT_MODULES_ENABLE="$TOMCAT_MODULES_ENABLE,gcp"
 fi
 
+if [ -n "$GCLOUD_PROJECT" ]; then
+  TOMCAT_PROPERTIES="gcp.project=${GCLOUD_PROJECT},${TOMCAT_PROPERTIES}"
```

Comment:
The properties is inserted before `TOMCAT_PROPERTIES` therefore if `gcp.project` is present into `TOMCAT_PROPERTIES` it will take the precedence over `$GCLOUD_PROJECT`.